README.md

```diff
@@ -238,16 +239,17 @@ Missing something? Write and publish your own webpack blocks!
 <details>
 <summary>How to debug?</summary>
 
-In case the webpack configuration does not work as expected you can debug it using [stringify-object](https://www.npmjs.com/package/stringify-object):
+In case the webpack configuration does not work as expected you can debug it using the [inspect](./packages/webpack#inspect) method:
 
 ```js
-const stringify = require('stringify-object')
+const { createConfig, inspect } = require('webpack-blocks')
 
 module.exports = createConfig([
   // ...
 ])
 
-console.log(stringify(module.exports))
+inspect(module.exports) // Print the whole config
+inspect(module.exports, 'module.rules') // Print part of the config
```

Comment:
Why would I want to do `inspect(module.exports, 'module.rules')` instead of `inspect(module.exports.module.rules)`? The only reason I can think of is if it throws a more meaningful error rather than the default one if `modules` is `undefined` or `null` for example.

Reply:
You don’t have to do it ;-)

Reply:
[It will return `undefined`](https://github.com/developit/dlv) if path doesn't exist.

Reply:
You can use `inspect(module.exports.module.rules)` if you like ;-)

Reply:
Sure but then if `module` is `undefined` it will throw. 👆